lib/tasks/update_lineagedb.rake

```diff
@@ -1,30 +1,67 @@
 require 'open3'
 require 'csv'
 require 'English'
-task update_lineage_db: :environment do
-  ### Usage: REFERENCE_S3_FOLDER=s3://idseq-database/taxonomy/2018-12-01 LINEAGE_VERSION=3 rake update_lineage_db
+
+desc 'Imports NCBI lineage data into IDseq'
+
+task 'update_lineage_db', [:dryrun] => :environment do |_t, args|
+  ### Short Usage: NCBI_DATE=2018-12-01 rake update_lineage_db
+  ### Full Usage: REFERENCE_S3_FOLDER=s3://idseq-database/taxonomy/2018-12-01 LINEAGE_VERSION=3 rake update_lineage_db
   ### REFERENCE_S3_FOLDER needs to contain names.csv.gz and taxid-lineages.csv.gz
   ### LINEAGE_VERSION needs to be incremented by 1 from the current highest version in taxon_lineages
+  current_date = Time.now.utc
+
+  puts "\n\nDRY RUN" if args.dryrun
+
+  ncbi_date = ENV['NCBI_DATE']
+  reference_s3_path = if ncbi_date.present?
+                        "s3://idseq-database/taxonomy/#{ncbi_date}"
+                      else
+                        ENV['REFERENCE_S3_FOLDER'].gsub(%r{([/]*$)}, '') # trim any trailing '/'
+                      end
+
+  puts "\n\nStarting import of #{reference_s3_path} ...\n\n"
+  import_lineage_database!(reference_s3_path, current_date) unless args.dryrun
+  puts "\n\nDone import of #{reference_s3_path}."
 
-  reference_s3_path = ENV['REFERENCE_S3_FOLDER'].gsub(%r{([/]*$)}, '') # trim any trailing '/'
   current_lineage_version = ENV['LINEAGE_VERSION'].to_i
+  if current_lineage_version.zero?
+    current_lineage_version = AlignmentConfig.last.lineage_version + 1
```

Comment:
We should use `AlignmentConfig.max("lineage_version")` here rather than `AlignmentConfig.last.lineage_version` to guarantee it's an increment. 

`AlignmentConfig.last` could be a revert of a previously generated AlignmentConfig with higher lineage_version, for example. We shouldn't introduce a requirement that the last AlignmentConfig is the one currently in use or the one with the most recent NCBI data

Reply:
Good point. I was just following what was in the guide.